extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -709,7 +709,7 @@
 #
 # Example: If you have a property named 'Age' which is of type 'Number' then add in LocalSettings:
 #
-# 	$GLOBALS['smwgFixedProperties'] = array(
+# 	$smwDefaults['smwgFixedProperties'] = array(
 #		'Age' => SMWDataItem::TYPE_NUMBER
 #  	);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)